cou/apps/base.py

```diff
@@ -236,13 +241,15 @@ def origin_setting(self) -> Optional[str]:
         return None
 
     @property
-    def possible_current_channels(self) -> list[str]:
-        """Return the possible current channels based on the current OpenStack release.
+    def possible_current_channel(self) -> str:
```

Comment:
:thinking: if there's only one is it still a "possible" channel? What are we trying to represent here? The channel the application should be using based on the workload version?

Reply:
> The channel the application should be using based on the workload version?

Yes. I think you are right on questioning this. Maybe changing it to `expected_current_channel` would be more appropriate